docs/src/styles/primitives/textAreaFieldStyles.css

```diff
@@ -0,0 +1,6 @@
+.globally-styled-textareafield.amplify-textareafield {
+  --amplify-components-fieldcontrol-border-color: rebeccapurple;
```

Comment:
Just think out loud here, we do have some inconsistencies on docs styling. We should decide on whether to use named colors or our own color palette

Reply:
Agreed, I can create a separate ticket to update all colors to use our amplify color variables.